client/layout/sidebar/style.scss

```diff
@@ -306,3 +306,35 @@
 		color: $blue-medium;
 	}
 }
+
+
+// Sidebar footer 
+.sidebar__footer {
+	position: absolute;
+	overflow: hidden;
+	padding: 0 6px;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	border-top: 1px solid lighten( $gray, 20% );
+
+	li {
```

Comment:
Try to avoid nesting here. See: [css guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#csssass-coding-guidelines)

We can pop this out of the .sidebar__footer if we add an additional class:
```
.sidebar__footer-item {
    display: inline;
}
```